package.json

```diff
@@ -30,7 +31,10 @@
     "open-source"
   ],
   "dependencies": {
-    "dotenv": "^16.4.4"
+    "@metamask/providers": "^17.2.0",
+    "daisyui": "^4.12.13",
+    "dotenv": "^16.4.4",
+    "viem": "^2.21.14"
```

Comment:
you should ask before using new libraries because we like to keep our projects unified. 
you can also look at the other pages we have to get an idea of which libraries we use

Reply:
> you should ask before using new libraries because we like to keep our projects unified. you can also look at the other pages we have to get an idea of which libraries we use

Alright!

Reply:
This includes tailwind, don't use it.